filament/src/ShadowMapManager.cpp

```diff
@@ -140,6 +151,7 @@ void ShadowMapManager::render(FrameGraph& fg, FEngine& engine, FView& view,
                         .width = mTextureRequirements.size, .height = mTextureRequirements.size,
                         .depth = 1,
                         .levels = 1,
+                        .samples = 1,
```

Comment:
This is what is causing the GL crash -- in fact this is always incorrect.  You're telling the framegraph you're requesting a depth buffer with a sample count of 1, but that's not true if you have MSAA.
By leaving the value to 0, you're telling the framegraph to "do the right thing" -- it'll automatically assign a sample count that matches the rendertarget's.

Now, I tried that, and I still get a crash in the GL driver, but I'm not sure why -- it looks  like a backend bug, I'm investigating.

Regardless I think setting samples=1 here is wrong, it should be either 0 or the requested msaa count.

Reply:
Actually I see what you attempted to do now :-)

You wanted to use a non MS texture so you can share it with all rendertargets regardless of their MSAA, relying on the "magic resolve"; i.e. the tile would have the right MS count, and the buffer would be discarded anyways. Man this needs a comment!

Very smart :-)

So basically the problem is that it's not supported in the GL backend right now -- but you're right it should be. I'm looking into it.